modules/utilities.test.js

```diff
@@ -3,16 +3,16 @@
 const chai = require('chai');
 const expect = chai.expect;
 
-const utilities = require('../../modules/utilities');
+const m = require('./utilities');
```

Comment:
Why `m`?

Reply:
Yeah, maybe it doesn't make too much sense here. I don't mind it as a convention for "the thing being tested" but it's a probably a bit silly as it's so terse, will change this back.